RubberduckTests/Refactoring/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -363,26 +365,26 @@ public void EncapsulatePrivateFieldInList()
             const string inputCode =
                 @"Private fi|zz As Integer, fuzz As Integer, fazz As Integer";
 
-            const string expectedCode =
-                @"
-Private fizz_1 As Integer, fuzz As Integer, fazz As Integer
+//            const string expectedCode =
```

Comment:
Commented out code?
Please remove or uncomment, in case this was meant to be commented out only temporarily.